java/jenv/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+
+github.setup    jenv jenv 0.5.4
+version         0.5.4
```

Comment:
There is no reason to set `version` again if you have already set it in `github.setup`.

Reply:
I've removed the `version` line. Forgot to remove that after I added the `github.setup` line indeed.